app/experimenter/nimbus-ui/src/components/Summary/EndEnrollment.tsx

```diff
@@ -0,0 +1,63 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React, { useCallback, useState } from "react";
+import { Alert, Button } from "react-bootstrap";
+
+const EndEnrollment = ({
+  onSubmit,
+  isLoading,
+}: {
+  isLoading: boolean;
+  onSubmit: () => void;
+}) => {
+  const [showEndConfirmation, setShowEndConfirmation] = useState(false);
+
+  const toggleShowEndConfirmation = useCallback(
+    () => setShowEndConfirmation(!showEndConfirmation),
+    [showEndConfirmation, setShowEndConfirmation],
+  );
+
+  return (
+    <div className="mb-4" data-testid="enrollment-end">
```

Comment:
Could be worth a follow-up issue, e.g. to combine these two buttons into a component side-by-side with state to display the confirmation form mutually exclusively for one or the other. Seems a decent enough chunk of work to avoid making this PR bigger